generated/nidmm/nidmm/_grpc_stub_interpreter.py

```diff
@@ -51,7 +51,7 @@ def _invoke(self, func, request):
         if grpc_error == grpc.StatusCode.INVALID_ARGUMENT:
             raise ValueError(error_message)
```

Comment:
It would seem these 2 (NOT_FOUND and INVALID_ARGUMENT) should be checked in the same if/elif block. Each raises an error based on `StatusCode`. (Existing code, but I'm evaluating the whole method since it's touched.)

Reply:
Grouped all uses of `grpc_error`.